.circleci/config.yml

```diff
@@ -197,35 +197,26 @@ jobs:
           path: test/reports
 
   cross_compile_gem:
-    machine:
-      image: ubuntu-2004:current
+    parameters:
+      platform:
+        description: "Platform to compile the gem resources"
+        type: string
+
+    docker:
+      - image: "ghcr.io/rake-compiler/rake-compiler-dock-image:1.3.0-mri-<< parameters.platform >>"
```

Comment:
Switching the base image here is a little surprising, can you share more why this is needed? Also, rather than pinning to 1.3.0, is there a tag for the latest release?

Reply:
> Switching the base image here is a little surprising, can you share more why this is needed?

It's been a while since I implemented this. But basically, we previously had this one Ubuntu job which compiled the ports for `x86-mingw32`, `x64-mingw32`. Now with this PR, we need to introduce another platform with `x64-mingw-ucrt`.

To speed to process up, I instead opt to run a separate job for each architecture. I do not recall how much time it saves, but I assume we talk about ~10 minutes per pipeline run.

> Also, rather than pinning to 1.3.0, is there a tag for the latest release?

Looking into [GitHub packages](https://github.com/rake-compiler/rake-compiler-dock/pkgs/container/rake-compiler-dock-image), there is a snapshot version available, which might be unstable. Then 1.3.0 is already the latest release.

I think it makes sense to pin this version to align it with the `rake-compiler-dock` gem.